### DISCUSSION TITLE
installing frePPLe extension module for Python

### DISCUSSION THREAD

**Discussion by makarov-m** [frePPLe/frepple#417] [General]
Hello,
I'm trying to launch frePPLE software v. 6.15 without a user interface. To do that I need to install a frePPLe library for Python as I assume. I found the documentation here https://frepple.org/docs/6.10/installation-guide/python-module.php how I can install it, however, I do not understand where to find **`frepple.pyd`** file (version for Windows), which is described at the first point of the installation guide. I also tried to find this file here https://github.com/frePPLe/frepple, locally and other repositories, but I was failed. It would be very helpful if somebody knows how to install this library. The installation guide for the 6.15 version of frePPLe is located here https://frepple.org/docs/6.15/installation-guide/index.html.

Thanks.

**Comment by jdetaeye** [comment]
>I'm trying to launch frePPLE software v. 6.15 without a user interface.

On the command line you'll run:     "frepple  your_python_script.py"

The runs the planning engine, and the embedded python interpreter executes your script.
Your script is responsible for a) reading data from your systems, b) creating frepple objects in memory, c) calling the solver and d) exporting the plan results to your systems.  The API isn't documented, but the folder https://github.com/frePPLe/frepple/tree/master/freppledb/input/commands gives an inspiring example.  Happy coding :-)  

>To do that I need to install a frePPLe library for Python as I assume. I found the documentation here https://frepple.org/docs/6.10/installation-guide/python-module.php how I can install it, however, I do not understand where to find frepple.pyd file (version for Windows), which is described at the first point of the installation guide.

The packaging of frepple as a python module has been removed since. The frepple.pyd file is no longer available.

The method described above provides the same capabilities.
